File: Cpp/src/general/fizz_buzz_calculator.cpp
#include <iostream>
#include "fizz_buzz_calculator.h"

using namespace std;

vector<string> FizzBuzzCalculator::FizzBuzz(int input) {
    vector<string> result;
    for (int i = 1; i <= input; i++) {
        if (i % 3 == 0) {
            if (i % 5 == 0) {
                result.emplace_back("FizzBuzz");
            } else {
                result.emplace_back("Fizz");
            }
        } else if (i % 5 == 0) {
            result.emplace_back("Buzz");
        } else {
            result.emplace_back(to_string(i));
        }
    }
    return result;
}

File: Python/src/general/fizz_buzz_calculator.py
from typing import List


class FizzBuzzCalculator:

    def fizz_buzz(self, data: int) -> List[str]:
        result = []
        for i in range(1, data+1):
            if i % 3 == 0:
                if i % 5 == 0:
                    result.append("FizzBuzz")
                else:
                    result.append("Fizz")
            elif i % 5 == 0:
                result.append("Buzz")
            else:
                result.append(str(i))
        return result
